phf_codegen/src/lib.rs

```diff
@@ -172,12 +180,23 @@ impl<K: Hash+PhfHash+Eq+fmt::Debug> Map<K> {
 /// A builder for the `phf::Set` type.
 pub struct Set<T> {
     map: Map<T>,
+    path: String,
```

Comment:
You can probably just look at the `path` of the `Map` instead of also tracking this here.